planetarium/NineChronicles
# Add 'Play to Earn' Banner.

Do not merge before the web page is opened/published.

### Description
SSIA

### How to test
1. Go to main menu.
2. Check if clicking event page proceeds into a url in this format.
`https://onboarding.nine-chronicles.com/nc-address?prefill="Current user's agent address"`

### Required Reviewers
@planetarium/9c-dev and @Namyujeong 



┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1141562434100787/1200503121969477) by [Unito](https://www.unito.io)
